### PR TITLE
Add configurable IMAP username prefix

### PR DESCRIPTION
**Pull request by herpiko**
Assign the prefix value in `conf/prod/surelia.json`, at `imapUsernamePrefix`.

### CODE REVIEW

**Comment by emhaye** [comment]
Ok :+1:

**Comment by emhaye** [comment]
Ok :+1:

File: src/app/controllers/surelia.js [hunk 1]
```diff
@@ -274,6 +275,11 @@ var Surelia = function ($scope, $rootScope, $state, $window, $stateParams, local
       return results;
     }
     var suggest_email_delimited = function(term) {
+      if (term.replace(/ /g,'')[term.length - 1] === ',') {
+        return self.nextSuggestion = false;
+      } else {
+        self.nextSuggestion = true;
+      }
       if (self.enableAutocomplete) {
```
Comment (mdamt) [inline src/app/controllers/surelia.js:283]: It would never reach here?

Comment (herpiko) [inline src/app/controllers/surelia.js:283]: Only if the last key stroke is equal to `,`, otherwise, it would reach there. Email suggestion still works.

Comment (herpiko) [inline src/app/controllers/surelia.js:283]: `nextSuggestion` controls wether the suggestion popup is shown or not.

**Pull request merged**